cmake/herbert_FindGTest.cmake

```diff
@@ -59,6 +60,8 @@ if(RESULT)
     message(FATAL_ERROR "Build step for googletest failed: ${RESULT}")
 endif()
 
+# We don't want to install GTest libs when we run `make install`
+set(INSTALL_GTEST OFF CACHE BOOL "Enable installation of googletest." FORCE)
```

Comment:
isn't this already covered by the 
```
if(${BUILD_TESTS})
   include(herbert_FindGTest)
   enable_testing()
endif()
```
block in the top-level CMake file?

Reply:
This flag disables the `install` target being created for GTest. If we did not have it, when we run `make install` CMake will copy all the GTest libraries/public headers into our package.